### PR TITLE
Add introspection_msgs

### PR DESCRIPTION
**Pull request by vmayoral**
These messages are used in rosnode list functionality.

### CODE REVIEW

File: introspection_msgs/msg/ROSMeta.msg [hunk 1]
```diff
@@ -0,0 +1,2 @@
+string node_name 	# ROS node name
+uint8[] id 			# DDS ID
```
Comment (tfoote) [inline introspection_msgs/msg/ROSMeta.msg]: Can you add some documentation to this message? A high level description would be helpful. And in particular why is the `id` field a vector? The short comment suggests that it's one ID.

And the name is not descriptive. Such that you only get a sense of the fact that it has metadata, but it clearly does not have all the metadata. It looks right now to be a name to id mapping.

**Comment by wjwwood** [comment]
One general comment is that it might make sense for these message to live in the `rcl_interfaces` package.

**Comment by vmayoral** [comment]
Thanks @tfoote and @wjwwood for the comments. 

Representing the `id` as a `string` instead of a `uint8[]` makes sense. With regard the naming and locations, I don't have a strong opinion about it.

I've made an attempt to document a bit more the message at https://github.com/erlerobot/common_interfaces/commit/dcd9fa517c9e008e87943eebff84f6c8be34d728. Feel free to cherry-pick the changes if you find it useful.

**Comment by mikaelarguedas** [comment]
closing this in the favor of the command line tools implemented in the [ros2cli](https://github.com/ros2/ros2cli/) repository. thanks @vmayoral for all your hard work!

**Pull request closed without merging**